io.catenax.iot_sensor_data/1.0.0/IoTSensorData.ttl

```diff
@@ -0,0 +1,275 @@
+] [
+  bamm:property :acceleration;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :weight;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :pressure;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :binarySensor;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :soundSensor;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :transmissionMethod;
+  bamm:optional "true"^^xsd:boolean
+] [
+  bamm:property :analogSensor;
+  bamm:optional "true"^^xsd:boolean
+] :catenaXId);
+    bamm:operations ();
+    bamm:events ();
+    bamm:preferredName "IoT Sensor Data"@en;
+    bamm:description "The data collected by an IoT Sensor Device"@en.
+:geoData a bamm:Property;
+    bamm:preferredName "Geo Data"@en;
+    bamm:description "Geodata, geographic data or geospatial data, refers to data and information that has explicit or implicit association with a location relative to Earth"@en;
+    bamm:characteristic :GeoDataCharacteristic.
+:batteryLevel a bamm:Property;
+    bamm:preferredName "Battery Level"@en;
+    bamm:description "The battery level displays how much charge of the battery has been left."@en;
+    bamm:characteristic :BatteryLevelCharacteristic;
+    bamm:exampleValue "50"^^xsd:decimal.
+:humidity a bamm:Property;
+    bamm:preferredName "Humidity"@en;
+    bamm:description "describes the concentration of water vapor present in the air."@en;
+    bamm:characteristic :HumidityCharacteristic;
+    bamm:exampleValue "74"^^xsd:decimal.
+:fillingLevel a bamm:Property;
+    bamm:preferredName "Filling Level"@en;
+    bamm:description "Volume percentage of a liquid or solid filling material in a container or vessel"@en;
+    bamm:characteristic :FillingLevelCharacteristic;
+    bamm:exampleValue "45"^^xsd:decimal;
+    bamm:see <https://www.giessereilexikon.com/en/foundry-lexicon/Encyclopedia/show/filling-level-4695/?cHash=25ad834655406130e927973c5b7d5f8b>.
+:temperature a bamm:Property;
+    bamm:preferredName "Temperature"@en;
+    bamm:description "physical quantity that expresses quantitatively the perceptions of hotness and coldness.?"@en;
+    bamm:characteristic :TemperatureCharacteristic;
+    bamm:exampleValue "25"^^xsd:decimal.
+:timestampSensoric a bamm:Property;
+    bamm:preferredName "Timestamp Sensoric"@en;
+    bamm:description "The timestamp of the latest sensor reading"@en;
+    bamm:characteristic bamm-c:Timestamp;
+    bamm:exampleValue "2023-02-05T15:48:54"^^xsd:dateTime.
+:light a bamm:Property;
+    bamm:preferredName "Light"@en;
+    bamm:description "The measure of brightness of a light source in terms of energy being emitted"@en;
+    bamm:characteristic :LightCharacteristic;
+    bamm:exampleValue "450.00"^^xsd:decimal;
+    bamm:see <https://www.wikidata.org/wiki/Q107780>.
+:acceleration a bamm:Property;
+    bamm:preferredName "Acceleration"@en;
+    bamm:description "the rate of change of the velocity of an object with respect to time"@en;
```

Comment:
uppercase and full stop. Please check all other descritption if the also have this problem.

Reply:
done